datasets/dataset_wrapper.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+from sklearn.model_selection import train_test_split
+"""
+Dataset Wrapper API 
+
+Provides some functionalities to access the datasets as a pandas DataFrame object.
+You can access the raw dataset as well as the the feature and label columns.
+You can also split the dataset into train and test datasets, and access them.
+
+"""
+class DatasetWrapper(object):
+    _dataset= None
+    X,y = None, None
+    X_train, X_test, y_train, y_test = None, None, None, None
+
+    def __init__(self,dataset_file:str) -> None:
+        """
+        Description:
+            To initialize a dataset object using the link found in the datasets folder 
+
+        Args:
+            test_dataset_name (string): a string that is the name of one of the test/datasets
+        """
+        
+        try:
+            # read the dataset from file path
+            self._dataset = pd.read_csv(dataset_file)
+
+            # set X,y
+            self.X,self.y = self._dataset.iloc[:,:-1], self._dataset.iloc[:,-1]
```

Comment:
I think we should pass in the label column name as a string to the function, and use that to identify the X and y parts. That way we don't need to make assumptions about column positions. We will want to use this class to handle all datasets interacting with PRESC, so we should stay general.

Reply:
Changed the function to include a label parameter. Also added a label setter function.

Reply:
Added a `label` parameter for this function and added a label setter function as well.